packages/terra-dropdown-button/package.json

```diff
@@ -42,6 +42,6 @@
     "precompile": "rm -rf lib",
     "test": "npm run test:jest && npm run test:wdio",
     "test:jest": "jest --config ../../jestconfig.js",
-    "test:wdio": "../../node_modules/.bin/wdio ../../node_modules/terra-dev-site/config/wdio/wdio.conf.js"
+    "test:wdio": "cd ../.. && npx wdio ./packages/$npm_package_name/wdio.conf.js"
```

Comment:
I haven't seen this before, are other components doing this?

Reply:
Yes other components are doing this, as in [here](https://github.com/cerner/terra-core/blob/6e84583b6841055f49bd4a52ebeb23a378a43384/packages/terra-button/package.json#L31).